docs/_quarto.yml

```diff
@@ -309,6 +267,37 @@ quartodoc:
             - Value
             - Column
             - Scalar
+            - name: literal
+              package: ibis
+              dynamic: true
+            - name: param
+              package: ibis
+              dynamic: true
+            - name: NA
+              package: ibis.expr.api
+            - name: "null"
+              package: ibis
+              dynamic: true
+            - name: coalesce
+              package: ibis.expr.api
+            - name: asc
+              package: ibis
+              dynamic: true
+            - name: desc
+              package: ibis
+              dynamic: true
+            - name: case
+              package: ibis
+              dynamic: true
+            - name: show_sql
+              package: ibis
+              dynamic: true
+            - name: to_sql
+              package: ibis
+              dynamic: true
+            - name: get_backend
```

Comment:
It looks like this documented in two places: once in `expression-generic` and once in `connection`.

Is that intentional?

Reply:
Yes. I think this is a functionality that is non-obvious to people, they aren't going to be looking for it. SQL, pandas, Polars, etc have no notion of this. So, I wanted to make it discoverable.

ifelse, on the other hand, people expect to exist and so are going to be looking for it, so that doesn't need to be advertised as much.

I considered the danger of these two places getting out of sync, but I think that is a low chance/impossible, since this is mostly a pointer to the docstring that actually has the content

Reply:
Generally not in favor of duplicating API documentation like this. I think there are better ways to advertise the API without duplicating its docs, like a blog post or a how-to.